ewah/operators/base.py

```diff
@@ -220,12 +221,29 @@ def base_init(
         )
         assert self._ACCEPTED_EXTRACT_STRATEGIES.get(extract_strategy), _msg
 
+        if isinstance(primary_key, str):
+            primary_key = [primary_key]
+
         if isinstance(hash_columns, str):
             hash_columns = [hash_columns]
 
+        if include_columns and isinstance(include_columns, str):
+            include_columns = [include_columns]
+            for col in primary_key:
+                if not col in include_columns:
+                    _msg = """
+                        Primary key {0} is not in the include_columns list.
+                        Make sure all primary keys are included.
+                        """.format(col)
+                    raise Exception(_msg)
```

Comment:
Syntax (blocking): This is not executed if `include_columns` is not a `str`. Please make sure it is executed regardless of `include_columns` type.

Reply:
Additionally, it will fail if `primary_key is None`. Make sure this case is also covered.